### ISSUE TITLE
`vmware.sh upload_ova`: ERROR: NOT_ALLOWED_IN_CURRENT_STATE

### ISSUE THREAD

**Issue by eugene-marchanka** [siderolabs/talos#7651]
### Description

When I run `upload_ova` for `v1.4.7` or `v1.4.8` everything works great but `vmware.sh` script fails when `upload_ova` for Talos `v1.5.0`:

```
 ./vmware.sh upload_ova
1f27e21a-89e6-450e-bd6f-d36f6e0fe057
[21-08-23 15:49:19] Uploading vmware-amd64.ovf... OK
[21-08-23 15:49:33] Uploading vmware-amd64.vmdk... OK
govc: 400 Bad Request: {"type":"com.vmware.vapi.std.errors.not_allowed_in_current_state","value":{"error_type":"NOT_ALLOWED_IN_CURRENT_STATE","messages":[{"args":["complete","ERROR","Error transferring file vmware-amd64.vmdk to ds:///vmfs/volumes/641337ca-f337b7b3-b0dd-00620b24a3b0//contentlib-1f27e21a-89e6-450e-bd6f-d36f6e0fe057/7ef814ee-50a8-4143-b348-d9c87abbee25/vmware-amd64_dec23985-ac5c-45a8-bda9-20497da685fd.vmdk?serverId=07cd1723-c320-4353-a9ea-42e13d588b01. Reason: IO error during transfer of ds:///vmfs/volumes/641337ca-f337b7b3-b0dd-00620b24a3b0//contentlib-1f27e21a-89e6-450e-bd6f-d36f6e0fe057/7ef814ee-50a8-4143-b348-d9c87abbee25/vmware-amd64_dec23985-ac5c-45a8-bda9-20497da685fd.vmdk?serverId=07cd1723-c320-4353-a9ea-42e13d588b01: Pipe closed"],"default_message":"The operation complete can not be invoked when the import session is in the ERROR state. Reason: Error transferring file vmware-amd64.vmdk to ds:///vmfs/volumes/641337ca-f337b7b3-b0dd-00620b24a3b0//contentlib-1f27e21a-89e6-450e-bd6f-d36f6e0fe057/7ef814ee-50a8-4143-b348-d9c87abbee25/vmware-amd64_dec23985-ac5c-45a8-bda9-20497da685fd.vmdk?serverId=07cd1723-c320-4353-a9ea-42e13d588b01. Reason: IO error during transfer of ds:///vmfs/volumes/641337ca-f337b7b3-b0dd-00620b24a3b0//contentlib-1f27e21a-89e6-450e-bd6f-d36f6e0fe057/7ef814ee-50a8-4143-b348-d9c87abbee25/vmware-amd64_dec23985-ac5c-45a8-bda9-20497da685fd.vmdk?serverId=07cd1723-c320-4353-a9ea-42e13d588b01: Pipe closed","id":"com.vmware.vdcs.cls-main.update_session_invalid_state_for_operation_reason"}]}}

```

### Environment

- Talos: v1.5.0
- Platform: VMWare vSphere 8

**Comment by smira** [comment]
This was already reported, fix is coming for v1.5.1 #7627

**Comment by smira** [comment]
I'm not really familiar with VMWare to understand the problem, if you can give me some hints on this failure is, we can work it out.
